### DISCUSSION TITLE
Disable wallpaper theming?

### DISCUSSION THREAD

**Discussion by Antisune101** [nix-community/stylix#593] [Q&A] [answered]
Hi,
I was using [swww](https://github.com/LGFae/swww) to  set my wallpaper on Hyprland, but since I set up stylix it stopped working. The image I gave stylix in my config is currently set as the wallpaper, but I'm unable to change my wallpaper with `swww img`.

Is there a way to disable / override the stylix theming for wallpaper?

**Comment by danth** [comment, marked as answer]
This is not currently possible through Stylix, although you can override the options we set, for example:

```nix
{ lib, ... }:

{
  services.hyprpaper.enable = lib.mkForce false;
}
```

See #501 (specific to Hyprland), #442 and #200 (general) for more details

**Reply by Antisune101** [reply]
Many thanks :)
